### PR TITLE
MISUV-7864-include-status

### PR DESCRIPTION
**Pull request by james-king-hmrc**
include status in response

### CODE REVIEW

File: app/models/reportDeadlines/ReportDeadlineModel.scala [hunk 1]
```diff
@@ -14,38 +14,66 @@
...
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+   */
+
+  package models.reportDeadlines
+
+  import models.reportDeadlines.ObligationStatus.{Fulfilled, Open}
```
Comment (ian-hmrc) [inline app/models/reportDeadlines/ReportDeadlineModel.scala:35]: The indentation isn't quite right here. Could you reformat please?

Comment (james-king-hmrc) [reply app/models/reportDeadlines/ReportDeadlineModel.scala]: done

Comment (ian-hmrc) [reply app/models/reportDeadlines/ReportDeadlineModel.scala]: Cool.

**Review by ian-hmrc — APPROVED** [review]
LGTM

File: app/models/reportDeadlines/ReportDeadlineModel.scala [hunk 2]
```diff
@@ -14,20 +14,44 @@
  * limitations under the License.
  */
 
+/*
```
Comment (joesheridan) [inline app/models/reportDeadlines/ReportDeadlineModel.scala:17]: why do we have 2 license messages now?

Comment (james-king-hmrc) [reply app/models/reportDeadlines/ReportDeadlineModel.scala]: done

**Review by ian-hmrc — APPROVED** [review]
LGTM

**Pull request merged**